client/src/components/Tags/model.js

```diff
@@ -5,6 +5,8 @@
 
 import { keyedColorScheme } from "utils/color";
 
+// VALID TAG REGEX
+const VALID_TAG_RE = /([a-zA-Z0-9])+(.[a-zA-Z0-9]+)?(:[a-zA-Z0-9]+)?/;
```

Comment:
utf-8 characters are valid in tags and we've included screenshots of this in past release notes ( (💩 for microbiome ), can we make these work ?

Reply:
@mvdbeek Yeah, definitely this was just a first pass at what I knew should work with the intent to follow up later.